content-accelerator/latest/configure/autofile.md

```diff
@@ -17,7 +17,7 @@ The `EVERYONE` authority must have Consumer permissions on this folder in order
 * **rootPath** (String, required) – path to the root Alfresco folder where auto-filed content will be stored
 * **types** (List, required) – list of content types that the auto-file configuration applies to (must be in QName format, e.g. `{http://www.alfresco.org/model/content/1.0}content`)
 
-* **propertiesList** (List, optional) – list of metadata properties that determines the folder path that content will be auto-filed into. For example, if you wanted to file documents by 2 metadata fields, "Project Name" and "Document Type", you’d configure auto-filing for these two properties, and then new content would be filed into the folder, `/{Root Path}/{Project Name}/{Document Type}` (must be in QName format, e.g. `{http://www.alfresco.org/model/content/1.0}content`). Property values can be split by adding brackets to the end of declaration (e.g. `{http://www.alfresco.org/model/content/1.0}name[0,4]`). If the `cm:name` was originally "Content Name", the property used for filing will be "Cont". If properties list is empty or null, document will be autofiled into the rootPath.
+* **propertiesList** (List, optional) – list of metadata properties that determines the folder path that content will be auto-filed into. For example, if you wanted to file documents by 2 metadata fields, "Project Name" and "Document Type", you’d configure auto-filing for these two properties, and then new content would be filed into the folder, `/{Root Path}/{Project Name}/{Document Type}` (must be in QName format, e.g. `{http://www.alfresco.org/model/content/1.0}content`). Property values can be split by adding a substring transformer as a prefix followed by the indexes in the format `~0~3` to the end of the declaration (e.g. `xSubstringBetweenTransformer~{http://www.alfresco.org/model/content/1.0}name~0~4`). If the `cm:name` was originally "Content Name", the property used for filing will be "Cont". If the properties list is empty or null, document will be autofiled into the rootPath.
```

Comment:
Update -  **by the indexes in the format `~0~3`**  to  **by the indexes in the format `~0~4`**  for consistency with the trailing line.